.github/workflows/test.yml

```diff
@@ -1,13 +1,13 @@
 ---
 name: Tests
-on: [push]
+on: [push, pull_request]
 jobs:
   linux-tests:
     name: Python ${{matrix.python-version}} tests on Linux
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "pypy3"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10.0-rc.3", "pypy3"]
```

Comment:
Yesterday I struggled quite some time to get the name right for Github Action. You should use something like: `3.10.0-alpha - 3.10.0`. But then you should make sure tox understands you ok. You can take a look to this commit for some "inspiration": https://github.com/psycopg/psycopg/commit/ce86d8051dd532cd6664f7bfdda5a0e781ef0b6d

Reply:
I see.

Reply:
Hmm, but the alpha release is old... :thinking: 

https://www.python.org/dev/peps/pep-0619/#schedule

I guess you'd want to add the latest 3.10 on psycopg.

Reply:
That's a range: it should select the most up-to-date version within the range. It [picked up the rc2](https://github.com/psycopg/psycopg/runs/3760031705#step:5:11).

Reply:
Ah! Cool!

I'm trying this before: https://github.com/adamchainz/patchy/blob/main/.github/workflows/main.yml

If it doesn't work, I'll pursuit what you did.